pkg/codegen/go/gen.go

```diff
@@ -3125,27 +3117,13 @@ func (pkg *pkgContext) genResourceModule(w io.Writer) error {
 	}
 
 	fmt.Fprintf(w, "func init() {\n")
-	if topLevelModule {
-		fmt.Fprintf(w, "\tversion, _ := PkgVersion()\n")
-	} else {
-		// Some package names contain '-' characters, so grab the name from the base path, unless there is an alias
```

Comment:
Don't we need this dash logic still?

Reply:
Right ok, it didn't look like any of the tests changed because of this so not knowing any better I'll assume your right here :)